### ISSUE TITLE
Border radius not working

### ISSUE THREAD

**Issue by mineiro** [evansims/hyper-firewatch#2]
Hey @evansims!

First and foremost, thanks for a wonderful theme! It really looks awesome on my MBP.

I have been using it with Hyper `Version 1.3.3 (1.3.3.1754)` for a while and noticed that the main window border radius is not working.

<img width="1292" alt="screen shot 2017-08-26 at 18 53 51" src="https://user-images.githubusercontent.com/3123/29745403-f1215bc0-8a8f-11e7-8de0-1086401394bd.png">

Have you noticed it too?

I have tried disabling other plugins to see if they could affect the theme, but without success.

Best regards!

**Comment by evansims** [comment]
Good catch, thanks! It appears something has changed in recent builds that broke the radius. Will investigate and update

**Comment by evansims** [comment]
Actually, noticing your wallpaper, do you happen to be running High Sierra? I'm running it as well and I think we might be seeing a bug with Electron's Vibrancy support and the macOS beta build. I'll keep digging, though

**Comment by mineiro** [comment]
Yeah! I'm using the latest beta. Digging around, maybe [#8305](https://github.com/electron/electron/issues/8305) on [electron/electron](https://github.com/electron/electron) is correlated.

**Comment by evansims** [comment]
Yup appears to be something to do with that, as disabling Vibrancy in the theme magically restores the rounded borders. I'll keep this issue open until a solution is figured out!
